.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "5"
+  - "4"
+
+before_script:
+  - "npm i -g eslint"
+  - "npm i -g npm@3"
+  - "cd ./tests/dummy npm i"
```

Comment:
```
error: Trying to use unknown adapter, "sails-memory", in model `author`.
error: Are you sure that adapter is installed in this Sails app?
error: If you wrote a custom adapter with identity="sails-memory", it should be in this app's adapters directory.
error: Otherwise, if you're trying to use an adapter named `sails-memory`, please run `npm install sails-memory --save
```
if without ```cd ./tests/dummy npm i```